cmd/crictl/image.go

```diff
@@ -125,11 +125,7 @@ var listImageCommand = cli.Command{
 					printHeader = false
 					fmt.Fprintln(w, "IMAGE\tTAG\tIMAGE ID\tSIZE")
 				}
-				repoTags := "<none>"
-				if image.RepoTags != nil {
```

Comment:
We should change this to `if len(image.RepoTags) > 0`.

Reply:
Only this is insufficient. repoTagsPair[1] will still cause panic  when it is "<none>" or "tag1"